sw/device/silicon_creator/rom/e2e/empty_test.c

```diff
@@ -7,8 +7,11 @@
 #include "sw/device/lib/runtime/log.h"
 #include "sw/device/lib/testing/test_framework/ottf_main.h"
 #include "sw/device/silicon_creator/lib/drivers/lifecycle.h"
+#include "sw/device/silicon_creator/lib/drivers/otp.h"
 #include "sw/device/silicon_creator/lib/manifest_def.h"
 
+#include "otp_ctrl_regs.h"
```

Comment:
Nit: could you wrap this include with `ifdef EMPTY_TEST_MSG`? 

Reply:
This was a simple hack but is slowly getting out of hand like all simple hacks do :) I'm planning to handle this at link-time after we make more progress on e2e tests.

That being said, I don't think `#ifdef` is what we really want because different tests use different drivers so it wouldn't achieve include-what-you-use.

Reply:
Also, ack code smell :)